ci/bamboo/build-manylinux-prototype.sh

```diff
@@ -1,101 +0,0 @@
-#!/bin/bash
```

Comment:
I use these scripts outside of ci a lot and plan to continue. It's useful to validate my changes locally before pushing changes. Any developer that wants to test the manylinux build outside of ci would use these scripts.